corehq/apps/user_importer/importer.py

```diff
@@ -275,36 +275,55 @@ def get_location_from_site_code(site_code, location_cache):
         )
 
 
-def create_or_update_users_and_groups(domain, user_specs, upload_user, group_memoizer=None, update_progress=None):
-    ret = {"errors": [], "rows": []}
+DomainInfo = namedtuple('DomainInfo', ['validators', 'can_assign_locations', 'location_cache', 'roles_by_name'])
 
-    group_memoizer = group_memoizer or GroupMemoizer(domain)
-    group_memoizer.load_all()
 
-    current = 0
+def create_or_update_users_and_groups(upload_domain, user_specs, upload_user, group_memoizer=None, update_progress=None):
```

Comment:
It looks like you're now ignoring the `group_memoizer` argument, or am I missing something?

Reply:
Currently, yes. I was going to figure out how to keep it for the upload domain, but pushed before I had that done. It doesn't sound like it will be hard, so will look into that today.